README.md

```diff
@@ -15,7 +15,7 @@ The leaderboard website displays scores submitted by different players. It also
 
 ## Live Demo
 
-[Live Demo Link](https://modest-ptolemy-3e81f3.netlify.app/)
+[Live Demo Link](https://keen-hermann-3978d3.netlify.app/)
 
 
 ## Getting Started
```

Comment:
- [ ] Please add in the README how to get the app up and running locally step by step like
`npm install` 
`npm run build`
`npm run start` 

so that anyone can understand how to get it locally.

Reply:
Hello @DeliceLydia. Thank you for your comment. I added the steps to run the app locally as you can see in the image below
![Capture106](https://user-images.githubusercontent.com/60292761/131880672-2722bf5c-280d-4086-bc1c-5798f1ac5feb.PNG)